# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: image/VectorImage.cpp
# Commit: b72d1d6131cc
# Full Hash: b72d1d6131cce6845af6ffd6f83de7f993930961
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2019-08-26 09:42:55
# Description:
#   Bug 1576154 - Check mSVGDocumentWrapper when trying to report use counters; r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43256
# ==============================================================================

diff -r 929421783c5b -r b72d1d6131cc image/VectorImage.cpp
--- a/image/VectorImage.cpp	Sun Aug 25 23:22:11 2019 +0000
+++ b/image/VectorImage.cpp	Sun Aug 25 22:45:33 2019 +0000
@@ -1283,7 +1283,8 @@
     mProgressTracker->OnDiscard();
   }
 
-  if (Document* doc = mSVGDocumentWrapper->GetDocument()) {
+  if (Document* doc =
+          mSVGDocumentWrapper ? mSVGDocumentWrapper->GetDocument() : nullptr) {
     doc->ReportUseCounters();
   }
 
